privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -15,7 +15,7 @@ myApp.controller("tokenDetailController", function ($scope,
     $scope.selectedRealms = {};
     $scope.newUser = {user: "", realm: $scope.defaultRealm};
     $scope.loggedInUser = AuthFactory.getUser();
-    $scope.machinesPerPage = 5;
+    $scope.machinesPerPage = 15;
```

Comment:
I recommend not to increase this to 15!
In details views the normal list length is 5. Like view 5 tokens in the list in the users view.

The token details view might get too long. 
What is your use case with privacyIDEA and the machines, here?

Thanks a lot
Cornelius